custom.example.css

```
@@ -40,6 +40,7 @@ to{}
 .frameless .frame--layout-splitview-landscape{}
 .frame--layout-splitview-landscape .frame--image{}
 /* src/css/image.css */
+.polling-paused.more-info .image--metadata{}
```

Biome

[error] An empty block isn't allowed.

Consider removing the empty block or adding styles inside it.

(lint/suspicious/noEmptyBlock)